pipelines/jobs/configurations/jdk11u_pipeline_config.groovy

```diff
@@ -183,29 +183,35 @@ class Config11 {
                 arch                 : 'riscv64',
                 dockerImage          : [
                         'hotspot'    : 'adoptopenjdk/ubuntu2004_build_image:linux-riscv64',
+                        'temurin'    : 'adoptopenjdk/ubuntu2004_build_image:linux-riscv64',
                         'openj9'     : 'adoptopenjdk/centos6_build_image',
                         'bisheng'    : 'adoptopenjdk/centos6_build_image'
                 ],
                 dockerArgs           : [
-                        'hotspot'    : '--platform linux/riscv64'
+                        'hotspot'    : '--platform linux/riscv64',
+                        'temurin'    : '--platform linux/riscv64'
                 ],
                 crossCompile         : [
                         'hotspot'    : 'dockerhost-rise-ubuntu2204-aarch64-1',
+                        'temurin'    : 'dockerhost-rise-ubuntu2204-aarch64-1',
                         'openj9'     : 'x64',
                         'bisheng'    : 'x64'
                 ],
                 buildArgs            : [
                         'hotspot'    : '--create-sbom',
+                        'temurin'    : '--create-sbom',
```

Comment:
Should we remove the hotspot line from these while replacing with Temurin?

Reply:
I wasn't sure what the policy should be, happy to go either way

Reply:
I will remove then